misc-tools/dj_judgehost_cleanup.in

```diff
@@ -47,7 +47,7 @@ case $1 in
         cut -d ' ' -f 2 /proc/mounts | grep -E "^$JUDGINGDIR" | while IFS= read -r mountpoint; do
             umount "$mountpoint"
             echo "Unmounted $mountpoint"
-            sleep 0.1
+            sleep 1
```

Comment:
This must really not be set to a second or it will take forever when too many mountpoints are present.

Reply:
What suggestion do you have?
- POSIX seems to only allow integers
- the alternative of usleep needs to be installed extra. 
- If we change the shebang to bash the original `sleep 0.1` would work
- We can make a loop which either delays (dirty) or checks if the mount still exists (I assume thats the reason the sleep is there?)

Reply:
This has nothing to with the shell: `sleep` is not a POSIX (or bash) shell builtin, but just a program that is specified by POSIX, see https://pubs.opengroup.org/onlinepubs/9699919799/utilities/sleep.html#top. However, GNU sleep supports sub-second resolution just fine. Since our judging backend runs on Linux only, I don't see a problem, just ignore.